generators/service-postgre/templates/swift/instrumentation.swift

```diff
@@ -1,12 +1,12 @@
 import Foundation
 import Dispatch
 import LoggerAPI
+import CloudEnvironment
 import SwiftKueryPostgreSQL
 
-var postgreSQLConn: PostgreSQLConnection!
-
-func initializeServicePostgre() throws {
+func initializeServicePostgre(cloudEnv: CloudEnv) throws -> PostgreSQLConnection {
     // Load credentials for PostgreSQL db using CloudEnvironment
+    let postgreSQLConn: PostgreSQLConnection
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment.

Reply:
done